### DISCUSSION TITLE
Console I/O: What's 'Wrap' and 'Banner'?

### DISCUSSION THREAD

**Discussion by humdingerb** [Genio-The-Haiku-IDE/Genio#83] [Q&A] [answered]
Both are currently disabled, it appears, but for translations it would begood to know.
(BTW, consider using the macro B_TRANSLATE_COMMENT() for non-obvious things that may need an explanation if the translator doesn't have the devs' context.)

I imaging "Wrap" is line-wrapping?
No idea about "Banner" though...

**Comment by jackburton79** [comment, marked as answer]
Wrap is line wrapping in the consoleIO view.
Banner is a setting which toggle showing a... banner (a line like "------ build start ------") before and after the build output in the Build view

**Reply by humdingerb** [reply]
Thanks for clearing that up.
Maybe "Start/End banner in console" would be at least a tiny bit more descriptive in the Settings window.
